target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-mikrotik-5acd.dtsi

```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2020, Robert Marko <robimarko@gmail.com> */
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	model = "MikroTik 5acD";
```

Comment:
I think you could do without the ````model````and ````compatible```` in the .dtsi root node there.
(They will get overwritten anyway).

Reply:
thanks! removed it.